### PR TITLE
Reactive extension for MKLocalSearchRequest

### PR DESCRIPTION
**Pull request by Marcocanc**
Not sure if/how we should test this.

### CODE REVIEW

File: ReactiveMapKit/MKLocalSearchRequest.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+private let defaultLocalSearchError = NSError(domain: "org.reactivecocoa.ReactiveCocoa.Reactivity.MKLocalSearchRequest",
+											  code: 1,
+											  userInfo: nil)
+@available(tvOS 9.2, *)
+extension Reactive where Base: MKLocalSearchRequest {
+	/// A SignalProducer which performs an `MKLocalSearch`.
+	public var search: SignalProducer<MKLocalSearchResponse, AnyError> {
+		return SignalProducer {[base = self.base] observer, lifetime in
+			let search = MKLocalSearch(request: base)
+			search.start(completionHandler: { response, error in
```
Comment (andersio) [inline ReactiveMapKit/MKLocalSearchRequest.swift]: [Nit] Trailing closure.

File: ReactiveMapKit/MKLocalSearchRequest.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import ReactiveSwift
+import ReactiveCocoa
+import Result
+import MapKit
+
+#if os(iOS) || os(tvOS) || os(macOS)
+private let defaultLocalSearchError = NSError(domain: "org.reactivecocoa.ReactiveCocoa.Reactivity.MKLocalSearchRequest",
+											  code: 1,
```
Comment (andersio) [inline ReactiveMapKit/MKLocalSearchRequest.swift]: Indentation.

**Comment by Marcocanc** [comment]
I'll try to cause some errors and report back

**Comment by Marcocanc** [comment]
I just tried to do a search without connection and it forwarded the NSError of the NSURLErrorDomain. So it looks like we can't strong type it : /

**Pull request merged**